gravitee-apim-console-webui/src/entities/portal/portalSettings.ts

```diff
@@ -229,5 +229,15 @@ export interface PortalSettingsPortalNext {
     title?: string;
     subtitle?: string;
     enabled?: boolean;
+    primaryButton?: BannerButton;
+    secondaryButton?: BannerButton;
   };
 }
+
+export interface BannerButton {
+  enabled?: boolean;
```

Comment:
Please put them in the same order as in Java.
`
            private String label;

            private String target;

            private Boolean enabled;

            private String type;

            private String visibility;
`

Reply:
Hmm I'm not sure if the Java file is pertinent for the front. The front-end cares about what is sent from the back, but it can differ in format from the Java files.